src/helper.cc

```diff
@@ -825,6 +832,8 @@ helper::~helper()
     /* note, don't free id_name, it probably points to static memory */
 
     // TODO: if the queue is not empty it will leak Helper::Request's
+    // TODO: Use DBG_IMPORTANT (WARNING) during and after shutdown.
+    // TODO: Use (DBG_CRITICAL) ERROR or even BUG before shutting_down.
```

Comment:
Please remove these out-of-scope TODOs. We are already addressing them in another issue, and adding them here may cause merge conflicts.